docs/sources/developers/angular_deprecation/angular-plugins.md

```diff
@@ -680,7 +680,7 @@ Lack of recent activity in the [project repository](https://github.com/MarcusCal
 
 > **Migration available - potential alternative:** Grafana provides the native [Canvas]({{< relref "../../panels-visualizations/visualizations/canvas/" >}}) panel.
 
-> **Migration available - potential alternative:** another plugin exists which provides similar capabilities - [ACE.SVG](https://grafana.com/grafana/plugins/aceiot-svg-panel/). Note this panel has compatibility issues with Grafana versions 10.0.0-10.1.0.
+> **Migration available - potential alternative:** other plugins exist which provide similar capabilities - [HTML](https://grafana.com/grafana/plugins/gapit-htmlgraphics-panel/), [Dynamic Text](https://grafana.com/grafana/plugins/marcusolsson-dynamictext-panel/) and [ACE.SVG](https://grafana.com/grafana/plugins/aceiot-svg-panel/) - note this panel has compatibility issues with Grafana versions 10.0.0-10.1.0.
```

Comment:
```suggestion
> **Migration available - potential alternative:** other plugins exist which provide similar capabilities - [HTML](https://grafana.com/grafana/plugins/gapit-htmlgraphics-panel/), [Dynamic Text](https://grafana.com/grafana/plugins/marcusolsson-dynamictext-panel/) and [ACE.SVG](https://grafana.com/grafana/plugins/aceiot-svg-panel/). Note this panel has compatibility issues with Grafana versions 10.0.0-10.1.0.
```

Reply:
@josmperez the compatibility issue only applies to ACE.SVG, with this suggestion I think that becomes less clear. Any other alternatives?

Reply:
updated suggestion